vmdb/app/models/orchestration_stack.rb

```diff
@@ -18,6 +18,10 @@ class OrchestrationStack < ActiveRecord::Base
   has_many   :outputs,    :dependent => :destroy, :foreign_key => :stack_id, :class_name => "OrchestrationStackOutput"
   has_many   :resources,  :dependent => :destroy, :foreign_key => :stack_id, :class_name => "OrchestrationStackResource"
 
+  alias stack_parameters parameters
+  alias stack_outputs outputs
+  alias stack_resources resources
```

Comment:
@bzwei Why do we prefix the variables with `stack_` and not `orchestration_stack_`?  Because `stack_parameters` may be misleading, if that is what the user sees.